schema.sql

```diff
@@ -72,3 +72,11 @@ CREATE TABLE visits (
     FOREIGN KEY (vets_id)
     REFERENCES public.vets (id)
 )
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ALTER TABLE owners ALTER COLUMN age DROP NOT NULL;
+
+CREATE INDEX idx_animals_id ON visits(animals_id);
+CREATE INDEX idx_vets_id ON visits(vets_id);
```

Comment:
👏 Great job with the initial analysis and the inclusion of screenshots. However, there's a specific step to follow as per the project requirements:

- [ ] Before creating the index, the execution time is 2429ms, which is more than 1s. As stated in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=The%20following%20queries%20are%20taking%20too%20much%20time%20(1%20sec%20%3D%201000ms%20can%20be%20considered%20as%20too%20much%20time%20for%20database%20query).), The query after using the index screenshot should show how much time the query takes AFTER the improvements introduced, and the execution time should be significantly lower than 1 second (1000 ms).

- [ ] Please remember to run the query again and update your screenshots. 🙏

To meet this requirement, kindly follow the steps mentioned by the previous reviewer and update your screenshot accordingly. This will ensure that your analysis aligns perfectly with the project specifications. Keep up the good work! 👍📊🕐